src/search/ui/offer.controller.js

```diff
@@ -43,6 +43,8 @@ function OfferController(
       controller.isGodUser = permission;
     });
   controller.init = function () {
+    $scope.preCovidDate = new Date('04/15/2020');
```

Comment:
If we want to reuse this feature in the future.
It might be a good idea to get this date from the appconfig so we can change the date via the appconfig.